test/unittests/parser_test.cpp

```diff
@@ -257,6 +257,48 @@ TEST(parser, function_section_with_multiple_functions)
     EXPECT_EQ(module.funcsec[3], 0xff);
 }
 
+TEST(parser, table_single_min_limit)
+{
+    const auto section_contents =
+        bytes{} + uint8_t{0x01} + uint8_t{0x70} + uint8_t{0x00} + uint8_t{0x7f};
```

Comment:
```suggestion
        bytes{0x01, 0x70, 0x00, 0x7f};
```
Not sure what do you want to fix here exactly.

Reply:
I want to replace the entire style in one commit. Similar changes apply to half of the file.